cfgov/v1/signals.py

```diff
@@ -2,20 +2,26 @@
 
 from django.conf import settings
 from django.core.cache import cache
+from django.db.models.signals import post_save
 from django.dispatch import receiver
 
 from wagtail.signals import page_published, page_unpublished
 
+from cdntools.backends import AkamaiBackend
```

Comment:
I think so. It'd be something like:

```python
from wagtail.contrib.frontend_cache import get_backends
akamai_backend = next(get_backends(backends="akamai"))
```

I didn't change this mostly because I was trying to change as little as necessary, while moving code around.

Reply:
Replaced in [88fc13f](https://github.com/cfpb/consumerfinance.gov/pull/8522/commits/88fc13fe8d9ce04a98f286a53c52c34c313df7b6)